### PR TITLE
Updates to support AWS EC2 deployment

### PR DESCRIPTION
**Pull request by Kraig5**
Added dotenv to read some environment variables: 
PLAYER_HOST, CHATS_HOST, CAROUSEL_HOST, CHANNELS_HOST, PROXY_PORT

index.html has alternate src paths for the bundles. Not sure how to handle this nicely (i.e. need a dotenv for the html file!) Ideas welcome...

### CODE REVIEW

**Comment by Nguyephi** [comment]
lgtm

How do you do environment variables without dotenv module?

**Comment by Kraig5** [comment]
Updated from node-fetch to express-http-proxy. I had trouble getting the service JavaScript bundles (like main.js or bundle.js) with fetch. I solved it with express-http-proxy.

File: index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 require('dotenv').config({ debug: process.env.DEBUG });
+var proxy = require('express-http-proxy');
```
Comment (azdoolittle) [inline index.js]: I'll have to read up on this.

**Comment by Kraig5** [comment]
Corrected errors -- missing slashes before service bundles (like main.js or bundle.js). This error was not detected because in the proxy www folder there were symbolic links to the bundles. So the proxy route failed, but the local static server picked them up.

**Comment by Nguyephi** [comment]
error fix lgtm

**Pull request merged**